uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/form/form.core.js

```diff
@@ -2,9 +2,12 @@
 
 define([
   'common/angular',
+  'common/directives/help-text.directive',
   'leave-absences/leave-type-wizard/leave-type-wizard.core'
 ], function (angular) {
   angular.module('leave-type-wizard.form.core', [
```

Comment:
Nothing in particular with this line, just wanted to discuss your app structure that i noticed only now (apologies)

What is the need for a `leave-type-wizard.form` feature module? Can't we flatten the structure a bit and just have `leave-type-wizard`? I can't see the need for added complexity given that from all we now, the entire module _is_ the wizard form.

If then in the future for some reason this module grows in complexity, then we can create sub-features (see [LIFT](https://github.com/johnpapa/angular-styleguide/tree/master/a1#lift), but doing it now seems premature?

Reply:
Done.